orders/views.py

```diff
@@ -1,3 +1,42 @@
-from django.shortcuts import render
+import json, uuid
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from orders.models    import OrderStatus, Order, OrderItem
+from users.models     import User
+from products.models  import Product
+
+from users.utils      import user_decorator
+
+class CartView(View):
+    @user_decorator
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            user           = request.user
+            product        = data['product_id']
+            current_amount = 0
+
+            if not Product.objects.filter(id=product).exists():
+                return JsonResponse({'message':'PRODUCT_DOES_NOT_EXIST'}, status=400)
+
+            order, created = Order.objects.get_or_create(
```

Comment:
```suggestion
            order, is_created = Order.objects.get_or_create(
```
boolean을 갖는 변수는 보통 앞에 is_을 붙여줍니다.

Reply:
넵 수정하겠습니다!